src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -341,11 +352,17 @@ docker run -p:8080:8080 \
            eclipse/che-server:6.0.0
 ----
 
-*Run Che on Public IP Address*
+*Running Che on a public IP address*
+
+If you want the remote browser clients to connect to the Che server (as opposed to the local browser clients) and override the defaults that are detected, set the `CHE_IP` variable to the Docker host IP address that will have requests forwarded to the `che-server` container.
+
+Run an auto-detection algorithm within the che-server container to determine this IP. 
 
-If you want to have remote browser clients connect to the Che server (as opposed to local browser clients) and override the defaults that we detect, set `CHE_IP` to the Docker host IP address that will have requests forwarded to the `che-server` container.
+* If Docker is running on the `boot2docker` tool, this is usually the `eth1` interface. 
```

Comment:
* get rid of the ifs
* boot2docker is deprecated; file a bug to remove the mention